### PR TITLE
Allow importing PyPI packages on wasm

### PR DESCRIPTION
**Pull request by coolreader18**
cc @birkenfeld

image

### CODE REVIEW

**Comment by birkenfeld** [comment]
That's nifty!  For my purposes the package doesn't come from PyPI though, since it needs to reflect the current master branch.

**Comment by coolreader18** [comment]
Ah, it looked like in demo.js it was downloading from pypi. Regardless, this can be extended to fetch any wheel, so as long as a Pygments.whl is built for the master branch this can work with that easily.

**Comment by jhoobergs** [comment]
How far would this feature eventually reach? Would it for example be possible to load numpy?

**Comment by coolreader18** [comment]
@jhoobergs maybe, but we can't load numpy at all at the moment, so it would be dependent on how extension modules eventually work. The `_numpy` module might have to be pre-packaged into the binary, especially for something like wasm which doesn't have dynamically-loaded modules yet.

File: vm/src/stdlib/zlib.rs [hunk 1]
```diff
@@ -89,86 +113,118 @@ mod decl {
...
+            }
+        }
+        fn compress(self, level: Compression) -> Compress {
+            match self {
+                #[cfg(feature = "zlib")]
+                Self::Standard { header, wbits } => {
+                    Compress::new_with_window_bits(level, header, wbits)
+                }
+                #[cfg(not(feature = "zlib"))]
+                Self::Standard { header } => Compress::new(level, header),
```
Comment (youknowone) [inline vm/src/stdlib/zlib.rs:144]: if this one doesn't share condition with the others, is this needed to be placed between `feature  = "zlib"`?

**Review by youknowone — APPROVED** [review]

File: vm/src/stdlib/zlib.rs (hunk 1 above)
Comment (coolreader18) [reply vm/src/stdlib/zlib.rs]: I moved it above the `feature = "zlib"` branches.

**Pull request merged**